cmd/woodpecker-autoscaler/flags.go

```diff
@@ -44,6 +44,12 @@ var flags = []cli.Flag{
 		Usage:   "time an agent is allowed to start before it can be terminated again as duration string like 2h45m (https://pkg.go.dev/time#ParseDuration)",
 		EnvVars: []string{"WOODPECKER_AGENT_ALLOWED_STARTUP_TIME"},
 	},
+	&cli.StringFlag{
+		Name:    "agent-inactivity-timeout",
+		Value:   "10m",
+		Usage:   "time an agent is allowed to be inactive before it can be terminated as duration string like 2h45m (https://pkg.go.dev/maze.io/x/duration#ParseDuration)",
```

Comment:
```suggestion
		Usage:   "time an agent is allowed to be inactive before it can be terminated as duration string like 2h45m (https://pkg.go.dev/time#ParseDuration)",
```